.eslintignore

```diff
@@ -9,3 +9,4 @@ next.config.js
 postcss.config.js
 commitlint.config.js
 docs
+coverage
```

Comment:
I also had an unnecessary error on commit with coverage files, so this line is needed! Thank you for adding it